notify/README.md

```diff
@@ -0,0 +1,75 @@
+# notify
+
+Sends a simple Slack notification for the results of a CI build.
+
+## How to use
+
+### Inline
+
+If you use the action within a single job, it will automatically declare
+the status:
+
+```
+name: CI
+on: push
+
+jobs:
+  tests:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      ...
+      - uses: ably/actions/notify@main
+        if: always()
+        with:
+          webhook: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
I'm assuming inputs from secrets are treated as secret when passed into an action and aren't logged?

Reply:
I'm pretty sure GHA ensures all secrets are masked in output